gomint-server/src/main/java/io/gomint/server/entity/EntityPlayer.java

```diff
@@ -1748,6 +1748,8 @@ public boolean shouldBeSeen(EntityPlayer player) {
     }
 
     public boolean knowsChunk(ChunkAdapter adapter) {
+        if(adapter == null)
```

Comment:
In which case can this be null?

Reply:
I dont know, I got an error whilst testing

Reply:
Ok i will check, i doubt this has something todo with your changes